bin/env.template

```diff
@@ -1,2 +1,27 @@
-# Path to the Liquidator source code
-export LIQUIDATOR_SRC_PATH=
+# Path to the Eva repo. Only used to run on Linux.
+LIQUIDATOR_SRC_PATH=<EVA REPOSITORY DIRECTORY>
+
+# General Config
+RPC_URL=https://rpc.ironforge.network/mainnet?apiKey=<IRONFORGE API KEY>
+YELLOWSTONE_ENDPOINT=<ENDPOINT URL>
+YELLOWSTONE_X_TOKEN=<TOKEN>
+SIGNER_PUBKEY=<LIQUIDATOR WALLET PUBKEY>
+KEYPAIR_PATH=<LIQIDATOR WALLET FILE>
+COMPUTE_UNIT_PRICE_MICRO_LAMPORTS=1000
+COMPUTE_UNIT_LIMIT=600000
```

Comment:
Out of curiosity, I tried our "worst case" 16 position test with this many CU and it was a success, so I can confirm this enough for the worst case!